lib/generators/thredded/install/templates/initializer.rb

```diff
@@ -21,7 +21,7 @@
 # User avatar URL. Thredded uses Gravatar via the gravtastic gem by default.
 # Visit the gravtastic project - https://github.com/chrislloyd/gravtastic#usage
 # for further instructions on how to customize.
-Thredded.avatar_url = ->(_user, post) { post.gravatar_url(default: 'mm') }
+Thredded.avatar_url = ->(user) { Gravatar.src(user.email, 128, 'mm') }
```

Comment:
should `128` be configurable somewhere?  what if I want HUGE avatars on my site??